mag-part-1/curry/curry_exercises.js

```diff
@@ -6,25 +6,41 @@ var _ = require('ramda');
 //==============
 // Refactor to remove all arguments by partially applying the function
 
-var words = function(str) {
-  return split(' ', str);
-};
+// var words = function(str) {
+//   return _.split(' ', str);
+// };
+
+function splitHere(separator,str) {
+  return split(separator, str);
+}
+
+var words = _.partial(splitHere, ' ')
 
 // Exercise 1a
 //==============
 // Use map to make a new words fn that works on an array of strings.
 
-var sentences = undefined;
-
+var sentences = (arrayOfStrings) => {
+  return arrayOfStrings.map( function (element) {
+    return element.split(' ')
+  })
+}
```

Comment:
I think the callback of map can be the words function from above? But this works too.